libs/app-shell/src/lib/components/plugin-launcher/plugin-launcher.component.ts

```diff
@@ -123,8 +126,18 @@ export class PluginLauncherComponent implements OnChanges, AfterViewChecked {
         }
 
         if (_pluginModule.type === 'angular-ivy-component') {
-            this._ngModule = await this.compiler.compileModuleAsync(_module);
-            this._ngComponent = _remoteModule[(_pluginModule as AngularIvyComponentDescriptor).component];
+            const ngPluginModule = _pluginModule as AngularIvyComponentDescriptor;
+            // if provided module name and component name we bootstrap both
+            if (ngPluginModule.name && ngPluginModule.component) {
```

Comment:
@fkolar this fallback is based on convention rather than data structure, but should work anyway.
I will test it with demo and will give a note when it's ready to merge

Reply:
great !